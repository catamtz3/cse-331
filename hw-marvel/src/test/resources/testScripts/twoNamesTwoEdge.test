LoadGraph twoName twoNames.csv

FindPath twoName CAT RAT