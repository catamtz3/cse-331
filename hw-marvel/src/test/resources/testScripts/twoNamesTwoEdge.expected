loaded graph twoName

path from CAT to RAT:
CAT to RAT via A2